src/MAUI/Maui.Samples/Samples/Layers/DisplayOACollection/DisplayOACollection.xaml

```diff
@@ -4,19 +4,13 @@
              xmlns:cmp="clr-namespace:Microsoft.Maui.Controls.Compatibility;assembly=Microsoft.Maui.Controls"
              xmlns:esriUI="clr-namespace:Esri.ArcGISRuntime.Maui;assembly=Esri.ArcGISRuntime.Maui"
              xmlns:resources="clr-namespace:ArcGISRuntimeMaui.Resources">
-    <cmp:RelativeLayout>
-        <esriUI:MapView x:Name="MyMapView"
-                        BindingContext="{x:Reference Name=ResponsiveFormContainer}"
-                        Style="{StaticResource MapWithFormStyle}" />
-        <resources:ResponsiveFormContainer x:Name="ResponsiveFormContainer">
-            <StackLayout>
-                <Label Grid.Row="1"
-                       Margin="5"
-                       Text="Pan and zoom to see features." />
-                <ActivityIndicator x:Name="LoadingProgressBar"
-                                   Margin="5"
-                                   IsRunning="True" />
+    <Grid Style="{DynamicResource EsriSampleContainer}">
+        <esriUI:MapView x:Name="MyMapView" Style="{DynamicResource EsriSampleGeoView}" />
+        <Border Style="{DynamicResource EsriSampleControlPanel}">
+            <StackLayout HeightRequest="75" Spacing="5">
+                <Label Text="Pan and zoom to see features." />
+                <ActivityIndicator x:Name="LoadingProgressBar" IsRunning="True" />
```

Comment:
This is intentional so that the map doesnt resize when panning in horizontal or vertical layouts due to the reappearing and disappearing indicator. I put it in a fixed sized grid to reduce the severity of the issue.